src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -226,17 +521,90 @@ void SchwarzschildVariables<DataType>::operator()(
 template <typename DataType>
 void SchwarzschildVariables<DataType>::operator()(
     const gsl::not_null<tnsr::I<DataType, 3>*> shift_excess,
```

Comment:
(Optional) From reading the implementation, this seems to just be the shift. I see that there is a `shift_background` that is just initialized to 0. If the `_background` and `_excess` terminology is something people readily recognize and it's just me, ignore me, but it wasn't clear to me why we need the two values. If you think it's not overexplaining things, it might be helpful to add a comment for the `shift_excess` and `shift_background` functions saying what they are / why both are needed / that `shift_excess` is just the shift (if that's true).

Reply:
The shift decomposition is an implementation detail of the XCTS system. Added a comment and ref to https://spectre-code.org/structXcts_1_1Tags_1_1ShiftExcess.html.

Reply:
Ah, I see, then I think it's probably on me that I should've read more documentation first!